ui/src/components/common/DismissibleNotifications.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Button, Form } from 'semantic-ui-react'
+
+import { useLedger,  useParty, useStreamQuery } from '@daml/react'
+import { ContractId } from '@daml/types'
+import { MarketRole } from '@daml.js/da-marketplace/lib/Marketplace/Utils'
+import {
+    DismissibleNotification as DismissibleNotificationTemplate
+} from '@daml.js/da-marketplace/lib/Marketplace/Notification'
+import { makeContractInfo, DismissibleNotificationInfo } from './damlTypes'
+import { useRegistryLookup } from '../common/RegistryLookup'
+import NotificationComponent from '../common/Notification'
+import FormErrorHandled from '../common/FormErrorHandled'
```

Comment:
This component is in `common` already, so the import paths can be simplified

Reply:
(side note: I kinda wish we could customize the webpack config because then can we do import aliases as described here: https://webpack.js.org/configuration/resolve/

The reason we can't at the moment is because we bootstrapped the UI from create-daml-app, which uses create-react-app scripts to manage all the underlying configs. We could eject https://create-react-app.dev/docs/available-scripts#npm-run-eject and drop react-scripts in favor of custom webpack/babel/etcetc configs, but I'm not sure if import aliases is a convincing enough reason to do that :P )

Reply:
Ha, it does certainly look a lot nicer with the import aliases though.